android/app/src/main/AndroidManifest.xml

```diff
@@ -386,54 +387,62 @@
 
     <activity
         android:name="app.organicmaps.bookmarks.BookmarkCategoriesActivity"
-        android:configChanges="orientation|screenLayout|screenSize"
+        android:configChanges="uiMode|orientation|screenLayout|screenSize"
         android:label="@string/bookmarks_and_tracks"
         android:parentActivityName="app.organicmaps.MwmActivity"
         android:windowSoftInputMode="adjustResize" />
 
     <activity
         android:name="app.organicmaps.bookmarks.BookmarkListActivity"
-        android:configChanges="orientation|screenLayout|screenSize"
+        android:configChanges="uiMode|orientation|screenLayout|screenSize"
         android:label="@string/bookmarks"
         android:parentActivityName="app.organicmaps.bookmarks.BookmarkCategoriesActivity"
         android:windowSoftInputMode="adjustResize" />
 
     <activity
         android:name="app.organicmaps.editor.EditorActivity"
-        android:configChanges="orientation|screenLayout|screenSize"
+        android:configChanges="uiMode|orientation|screenLayout|screenSize"
         android:label="@string/edit_place"
         android:parentActivityName="app.organicmaps.MwmActivity"
         android:windowSoftInputMode="adjustResize" />
 
     <activity
         android:name="app.organicmaps.editor.ProfileActivity"
+        android:configChanges="uiMode"
         android:parentActivityName="app.organicmaps.settings.SettingsActivity" />
 
     <activity
         android:name="app.organicmaps.editor.FeatureCategoryActivity"
+        android:configChanges="uiMode"
         android:parentActivityName="app.organicmaps.MwmActivity"
         android:windowSoftInputMode="adjustResize" />
 
     <activity
         android:name="app.organicmaps.editor.ReportActivity"
+        android:configChanges="uiMode"
         android:parentActivityName="app.organicmaps.MwmActivity" />
 
     <activity
         android:name="app.organicmaps.editor.OsmLoginActivity"
+        android:configChanges="uiMode"
         android:parentActivityName="app.organicmaps.MwmActivity" />
 
     <activity
       android:name="app.organicmaps.bookmarks.BookmarkCategorySettingsActivity"
       android:label="@string/edit"
+      android:configChanges="uiMode"
       android:windowSoftInputMode="stateVisible"/>
     <activity
       android:name="app.organicmaps.widget.placepage.PlaceDescriptionActivity"
+      android:configChanges="uiMode"
```

Comment:
Is there a way to automatically add it to any activity without explicitly specifying everywhere?

Reply:
The XML attributes are only inherited by activities that actualy inherit (in java) other activities. So we were able to reduce the number of 'android:configChanges="uiMode"' by adding the BaseMwmFragmentActivity to the manifest and giving it that attribute.
However, any activity that specifies its configChanges, has to have "uiMode" set explicitly